models/token.js

```diff
@@ -0,0 +1,13 @@
+const jwt = require("jsonwebtoken");
+require("dotenv").config();
+
+const createToken = (id) => {
+  const payload = { id };
+  const secret = process.env.SECRET;
+  const token = jwt.sign(payload, secret, { expiresIn: "1h" });
```

Comment:
jeśli chodzi o testy to ja bym ustawił na kilka dni ważność tokena, bo inaczej co godzine musisz go zmieniać :)